docs/source/tagset/elemento-article.rst

```diff
@@ -126,7 +126,7 @@ Exemplo `JATS versão 1.1 <http://jats.nlm.nih.gov/publishing/1.1/>`_:
 
 .. code-block:: xml
 
-     <article xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML" dtd-version="1.1" specific-use="sps-1.8" article-type="research-article" xml:lang="pt">
+     <article xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML" dtd-version="1.1" specific-use="sps-1.9" article-type="research-article" xml:lang="pt">
```

Comment:
aqui não teria que ser sps-1.10?

Reply:
É só um exemplo Robs. Mas posso mudar se for preferir.

Reply:
Corrigido. Mas veja que já existe a informação de que:

.. note:: No atributo ``@specific-use`` o valor **sps-1.10** é apenas uma referência genérica à versão da *SciELO PS*. Deve ser utilizada uma das `versões vigentes <http://docs.scielo.org/projects/scielo-publishing-schema/pt_BR/latest/index.html#notas-da-versao>`_.